webpay/settings/base.py

```diff
@@ -633,3 +633,7 @@
 FXA_OAUTH_URL = 'https://oauth-stable.dev.lcip.org'
 FXA_CLIENT_ID = 'f69290928139452e'
 FXA_CLIENT_SECRET = 'f6d74bf347fe8dab38c0103b421ae12f276c47ba4914cf85b9927041667c3237'
+
+# Set of email addresses for users that gain super powers once authenticated.
+# For example, they will see an admin screen to simulate certain things.
+USERS_WITH_SUPER_POWERS = set([])
```

Comment:
I filed for the removal and was sure to add 'clean up' in the bug description for ya ;) https://bugzilla.mozilla.org/show_bug.cgi?id=1125444